frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/status-card.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { Gallery, GalleryItem } from '@patternfly/react-core';
+import { ALERTS_KEY } from '@console/internal/actions/dashboards';
+import AlertsBody from '@console/shared/src/components/dashboard/status-card/AlertsBody';
+import AlertItem from '@console/shared/src/components/dashboard/status-card/AlertItem';
+import { alertURL, PrometheusRulesResponse } from '@console/internal/components/monitoring';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import HealthBody from '@console/shared/src/components/dashboard/status-card/HealthBody';
+import HealthItem from '@console/shared/src/components/dashboard/status-card/HealthItem';
+import { getAlerts } from '@console/shared/src/components/dashboard/health-card/utils';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import {
+  DATA_RESILIENCY_QUERY,
```

Comment:
Shouldn't this be `DATA_RESILIENCY_QUERIES` (plural) ?

Reply:
It will be singular actually, changed below as well

Reply:
OK, thanks for clarification.

My understanding was that `<NAME>_QUERIES` constants in `queries.ts` files were, by convention, maps (query name to its value) which is why I suggested the plural.